### PR TITLE
Added dynamic configuration for CoreRestApi on Content side

### PR DESCRIPTION
**Pull request by dansterenson**
Should be merged after SDK release with the following PR: https://github.com/demisto/demisto-sdk/pull/2722

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-6112
https://jira-hq.paloaltonetworks.local/browse/CIAC-528

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ python = "^3.8,<3.11"
 defusedxml = "^0.7.1"
 
 [tool.poetry.group.dev.dependencies]
-demisto_sdk = "1.14.0"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev="789278337995c786e55b80f50c58d6410f70689b"}
```
Comment (DeanArbel) [inline pyproject.toml:13]: @dansterenson please confirm this action with @dorschw

Comment (dorschw) [reply pyproject.toml:13]: do not merge. let's discuss it offline

Comment (dansterenson) [reply pyproject.toml:13]: It was just for testing. As written, it should be merged after [this](https://github.com/demisto/demisto-sdk/pull/2722) is merged.

Comment (dansterenson) [reply pyproject.toml:13]: This file will be reverted once the SDK version is released.

**Comment by dansterenson** [comment]
@DeanArbel Fails on 
"[ERROR]: Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI.yml: [RM110] - The following commands appear in CoreRESTAPI.yml but not in the README file:
demisto-api-post
demisto-api-get
demisto-api-put
demisto-api-delete
demisto-api-download
demisto-api-multipart
demisto-delete-incidents
demisto-api-install-packs"

Force merge is needed.

**Pull request merged**